src/pyvesync/vesynckitchen.py

```
@@ -251,7 +251,7 @@ def status_response(self, return_status: dict) -> None:
         self.current_temp = return_status.get('curentTemp')
         self.cook_set_temp = return_status.get('targetTemp',
                                                return_status.get('cookSetTemp'))
-        self.temp_unit = return_status.get('tempUnit', self.temp_unit)  # Always keep set
+        self.temp_unit = str(return_status.get('tempUnit', self.temp_unit))  # Always keep set
```

Ruff (0.12.2)

Single quotes found but double quotes preferred

Replace single quotes with double quotes

(Q000)